tillerinobot/src/main/java/tillerino/tillerinobot/lang/TsundereGerman.java

```diff
@@ -0,0 +1,392 @@
+	}
+
+	StringShuffler optionalCommentOnNPHardShuffler = new StringShuffler(rnd);
+	StringShuffler optionalCommentOnNPEasyShuffler = new StringShuffler(rnd);
+
+	@Override
+	public Response optionalCommentOnNP(OsuApiUser apiUser, BeatmapMeta meta) {
+		if (Math.random() > 0.25) {
+			return new NoResponse();
+		}
+		PercentageEstimates estimates = meta.getEstimates();
+		double typicalPP = (apiUser.getPp() / 20.0);
+		if (estimates.getPPForAcc(.95) / typicalPP > 2.0) {
+			return new Message("Ist das dein Ernst!? Wenn dich diese Map nicht tötet werde ich es machen.");
+		} else if (estimates.getPPForAcc(1) / typicalPP < 0.333) {
+			return new Message("Diese Map zu spielen wird mich nicht sonderlich beeindrucken... n-n-nicht dass ich das wollen würde.");
+		}
+		return new NoResponse();
+	}
+	
+	@Override
+	public Response optionalCommentOnWith(OsuApiUser apiUser, BeatmapMeta meta) {
+		//The following checks are probably redundant, but they don't hurt anyone either.
+		if (Math.random() > 0.25) {
+			return new NoResponse();
+		}
+		PercentageEstimates estimates = meta.getEstimates();
+		double typicalPP = (apiUser.getPp() / 20);
+		if (estimates.getPPForAcc(.95) / typicalPP > 2.0) {
+			return new Message("Idiot! Du wirst dich noch verletzen wenn du diese Mods benutzt!");
+		} else if (estimates.getPPForAcc(1) / typicalPP < 0.5) {
+			return new Message("Wenn du wie ein Baby behandlt werden willst, hättest du einfach fragen müssen... nein, fang einfach an zu spielen.");
+		}
+		return new NoResponse();
+	}
+	
+	@Override
+	public Response optionalCommentOnRecommendation(OsuApiUser apiUser, Recommendation meta) {
+		setChanged(true);
+
+		recentRecommendations++;
+		if(recentRecommendations == 7) {
+			return new Message("Ich hab viel Freizeit. Ich würde nie Maps raussuchen weil ich dich mag... r-r-rein hypothetisch.");
+		} else if(recentRecommendations == 17) {
+			return new Message("Weißt du, es ist ein Privileg so viel mit mir zu reden, kein Recht.");
+		} else if(recentRecommendations == 37) {
+			return new Message("Wie würdest du eigentlich dieses Spiel spielen wenn ich dir nicht die ganze Zeit sagen würde wie?");
+		} else if(recentRecommendations == 73) {
+			return new Message("Ich hätte dich schon längst für Belästigung angezeigt wenn ich dich nicht lieb... Ich hab nichts gesagt.");
+		} else if(recentRecommendations == 173) {
+			return new Message("Kannst mich einfach nicht allein lassen, was? Ich d-denke das ist okay. Aber wag es nicht das jemandem zu erzählen!");
+		}
+		return new NoResponse();
+	}
+	
+	transient boolean changed;
+
+	@Override
+	public boolean isChanged() {
+		return changed;
+	}
+
+	@Override
+	public void setChanged(boolean changed) {
+		this.changed = changed;
+	}
+
+	StringShuffler invalidAccuracyShuffler = new StringShuffler(rnd);
+	
+	@Override
+	public String invalidAccuracy(String acc) {
+		setChanged(true);
+
+		return invalidAccuracyShuffler.get(
```

Comment:
All of these are quotes by famous people. I'd look over them again and maybe look for translations that other people have done, if you haven't already done so.